doc/releases/release-notes-3.2.rst

```diff
@@ -1395,6 +1400,15 @@ Libraries / Subsystems
 
 * Testsuite
 
+  * Added Kconfig support to ``unit_testing`` platform.
+  * Migrate tests to use :kconfig:option:`CONFIG_ZTEST_NEW_API`
+  * Add ztest options for shuffling tests/suites via:
+    * :kconfig:option:`CONFIG_ZTEST_SHUFFLE`
```

Comment:
I think that doxygen wants an emptyline before indentation changes, so one here and one after this block.

Reply:
Done